src/main/scala/sbtconsistent/SbtConsistentPlugin.scala

```diff
@@ -24,66 +23,67 @@ object SbtConsistentPlugin extends AutoPlugin {
         val scalaV = (update / scalaBinaryVersion).value
         val originalName = m.name
         val name = originalName.stripSuffix(s"_$sbtV").stripSuffix(s"_$scalaV")
-        m.withName(s"${name}_sbt1_2.12")
+        m.withName(s"${name}_${}")
```

Comment:
Issue here; noticed thanks to the github action test (test coverage is seriously essential generally) -- I think here you want to just include hardcoded suffix